conf/application.conf.template

```diff
@@ -67,6 +67,7 @@ security {
 # Ore configuration
 ore {
     date-format =  "MMM dd, yyyy"
+    date-and-time-format = "yyyy-MM-dd HH:mm"
```

Comment:
The date format should be the same as the above line.

Reply:
I don't think so. Sometimes you don't need time information (hours, minutes, seconds...), that's why I think it's fine to have a date format and a date format with time information.

Reply:
But they should be consistent, like:
```
    date-format =  "MMM dd, yyyy"
    date-and-time-format = "MMM dd, yyyy HH:mm"
```

Reply:
Oh yeah, I agree.